src/platform/user/profile/vet360/components/PhoneField/PhoneField.jsx

```diff
@@ -26,7 +26,7 @@ const formSchema = {
     },
     extension: {
       type: 'string',
-      pattern: '^\\s*[a-zA-Z0-9]{1,10}\\s*$',
+      pattern: '^\\s*[a-zA-Z0-9]{0,10}\\s*$',
```

Comment:
Needed to change the min characters from 1 to 0 or this field was effectively required and couldn't be left blank. 😱

Reply:
:confused: How was this working before? Are there any tests for this?

Reply:
It was working before, _but_ if you focused the field and didn't enter anything, it would fail the regex because an empty string isn't at least one number character.